build/ci/postInstall.js

```diff
@@ -76,5 +77,38 @@ function createJupyterKernelWithoutSerialization() {
     console.log(colors.green(destPath + ' file generated (by Jupyter VSC)'));
 }
 
-fixJupyterLabDTSFiles();
-createJupyterKernelWithoutSerialization();
+/**
+ * In order to generate random bytes on Windows without taking a dependency on native node modules
+ * which we then need to build xplat and bundle with the extension, download a prebuilt executable
+ * which directly consumes BCryptGenRandom in bcrypt.dll and outputs random bytes as hex. This
+ * executable is required for trusted notebooks key generation and is included with the built extension.
+ */
+async function downloadBCryptGenRandomExecutable() {
+    console.log('Downloading BCryptGenRandom.exe...');
+    const executableName = 'BCryptGenRandom.exe';
+    const uri = `https://pvsc.blob.core.windows.net/jupyter-dev-builds/${executableName}`;
+    const srcDestination = path.resolve(path.dirname(__dirname), '..', 'src', 'BCryptGenRandom');
+    const srcDestinationFilename = path.join(srcDestination, executableName);
+    if (fs.existsSync(srcDestinationFilename)) {
+        console.log('BCryptGenRandom.exe is already downloaded.');
+    } else {
+        fs.ensureDirSync(srcDestination);
```

Comment:
No need to download if `outDestinationFileName` already exists.
I think we should check if that file exists first.